cgi-bin/LJ/Talk.pm

```diff
@@ -1516,11 +1516,20 @@ sub talkform {
         'form_url' => LJ::create_url( '/talkpost_do', host => $LJ::DOMAIN_WEB ),
         'errors' => $opts->{errors},
         'create_link' => '',
+
         'editid' => $editid,
         'editreason' => $comment ? $comment->edit_reason : '',
+        'comment_body' => $form->{body} || '',
```

Comment:
Do you really mean `||` (truthy-or) or would `//` (defined-or) be better here?

Reply:
Now that I look closer, I think I don't actually want to bother with any "or," because TT will just output an empty string if I render an undefined value, which is what I want. (I dropped that unnecessary logic for a couple of later pieces, but not this one.)

(P. sure I just copied this from the existing logic, which I think predated `//` being reliably available.)